examples/sdkconfig.defaults

```diff
@@ -1,5 +1,5 @@
 # Rust often needs a bit of an extra main task stack size compared to C (the default is 3K)
-CONFIG_ESP_MAIN_TASK_STACK_SIZE=32768
+CONFIG_ESP_MAIN_TASK_STACK_SIZE=45768
```

Comment:
di you try to reduce it a little bit? I would like to limit stack usage as much as possible

Reply:
ok